subworkflows/metadata_and_phylotree.nf

```diff
@@ -5,6 +5,71 @@
 include { METADATA_TABLE } from '../modules/metadata_table'
 include { PHYLO_TREE } from '../modules/phylo_tree'
 
+process PREPARE_LOCATION_INPUT {
```

Comment:
I think you should create `utils.nf` in modules and move processes there. It overloads subwf